src/version.ts

```diff
@@ -1 +1 @@
-export const version = "0.1.8"; // TODO(@qu4k): ugly
+export const version = "0.1.9"; // TODO(@qu4k): ugly
```

Comment:
I think we should update version once the rewrite lands.

Reply:
We'll do another version after the rewrite.
Currently, no version of eggs works under Deno 1.2, we had to release this version as soon as possible.

Reply:
Understandable. Good to go! 
Besides I like 0.2.0 for the rewrite 🌟